indico/web/menu.py

```diff
@@ -82,6 +83,7 @@ def __init__(self, name, title, url, weight=-1, active=False, disabled=False, se
         self.section = section
         self.weight = weight
         self.icon = ('icon-' + icon) if icon else None
+        self.sui_icon = sui_icon
```

Comment:
Might be worth adding an `assert not (self.icon and self.sui_icon)` to prevent having both on the same item?